platform/how-to.mdx

```diff
@@ -1,14 +1,48 @@
 ---
-title: 'How-to'
+title: 'Use Replicant Network'
 description: 'How to use Replicant Network'
 icon: 'code'
 ---
 
-Replicant Network is a platform where you can:
+**Replicant Network** is a platform where you can:
 - request and create a bounty for a new custom model
 - participate in existing bounties
 - use custom AI models
+- earn rewards by owning commonly used models
 
 Models on the platform are tokenized and owned by users, meaning that users collectively participate in governance decisions about model evolution as well as receive a revenue share when the model is used.
 
-More information about the whole workflow can be found in the [Workflow](/platform/workflow) section.
\ No newline at end of file
+More information about the whole workflow can be found in the [Workflow](/platform/workflow) section.
+
+## Create a bounty
+You can create a bounty for a new model by going to the [Replicant Network](https://alpha.rplcnt.io/create-bounty). You will be asked to provide information about the model you want to create:
+- Title - name of your model.
+- Symbol - a short symbol for the token of your model.
+- Type - the type of model you want to create. Currently only `Image Generation` is supported.
+- Token Image - an image for the model that will be displayed on the platform.
+- Description - a detailed description of the model you want to create. The more details you provide, the easier it will be for the developer to create the model you want.
+- Type of Input - the type of input you want to provide to the model. Currently only `Text` is supported.
+- Base Model - the base model you want to use for your custom model. Currently only `Stable Diffusion` is supported.
+- Bounty Periods - timelines for the bounty. You need to set Funding period, Submissions period & Voting period. More details about the bounty phases can be found below.
+- Maximum Amount - the maximum amount of MON collected for the bounty. This should be set if you want to ensure you get the % of model share. For example if you set limit to 1000 MON and you contribute 250MON, you will get at least 25% of the model share.
+- Developer Fees - % of the total model supply that will go to developer.
+- Your Contribution - Amount you wish to contribute.
+
+### Bounty Phases
```

Comment:
this is repeated - maybe we should structure things so text is not repeated, so someone who is going through the docs doesn't have to read everything multiple times